assets/js/components/data-table.js

```diff
@@ -62,6 +59,7 @@ export const getDataTableFromData = ( data, headers, options ) => {
 			}
 
 			const hiddenOnMobile = options && options.hideColumns && options.hideColumns.mobile.includes( i );
+			const permaLink = link[ 0 ] === '/' ? global._googlesitekitLegacyData.admin.siteURL + link : link;
```

Comment:
`link` may be `undefined` here, which currently causes e2e tests to fail.
```suggestion
			const permaLink = link?.[ 0 ] === '/' ? global._googlesitekitLegacyData.admin.siteURL + link : link;
```

Reply:
Haven't seen this and updated it slightly differently.